spras/containers.py

```diff
@@ -148,6 +150,43 @@ def run_container(framework: str, container_suffix: str, command: List[str], vol
     else:
         raise ValueError(f'{framework} is not a recognized container framework. Choose "docker" or "singularity".')
```

Comment:
Not in scope, but I noticed this error message is wrong and should include "dsub" as a choice.

Reply:
If dsub is missing other places, it was an oversight in #195 and almost certainly not an intentional omission.